### ISSUE TITLE
Camera calibration when using gscam_decklink_stereo

### ISSUE THREAD

**Issue by Cartucho** [jhu-dvrk/dvrk-ros#32]
### Part 1

To calibrate the cameras in your instructions you start by telling us to run the following command:

`roslaunch dvrk_robot gscam_stereo.launch rig_name:=jhu_daVinci rect:=false`

However, since I am using the decklink I replaced it by the following command:

`roslaunch dvrk_robot gscam_decklink_stereo.launch rig_name:=patterson`

Note that in this case I did not pass the arg `rect:=false` since it doesn't exist in the file `gscam_decklink_stereo.launch`

---------------------------------------------------------

### Part 2

Then, I tuned the next command from:

`rosrun camera_calibration cameracalibrator.py --size 11x10 --square 0.005 right:=/jhu_daVinci/right/image_raw left:=/jhu_daVinci/left/image_raw right_camera:=/jhu_daVinci/right left_camera:=/jhu_daVinci/left --approximate=0.050`

to:

`rosrun camera_calibration cameracalibrator.py --size 11x10 --square 0.005 right:=/patterson/right/decklink/camera/image_raw left:=/patterson/left/decklink/camera/image_raw right_camera:=/patterson/right/decklink/camera left_camera:=/patterson/left/decklink/camera --approximate=0.050`

Basically I replaced `jhu_daVinci` by `patterson`, `right` by `right/decklink/camera` and `left` by `left/decklink/camera`.

However I get the following message:

![image](https://user-images.githubusercontent.com/15831541/58980271-d26b4d80-87c7-11e9-89b8-ff2b9946b879.png)

**Comment by Cartucho** [comment]
Adding the rostopic list to the description:

![image](https://user-images.githubusercontent.com/15831541/59776820-5f73d380-92ab-11e9-8fa9-0d4e45675a76.png)
